frontend/dockerfile/docs/reference.md

```diff
@@ -1509,12 +1463,59 @@ attempted to be used instead.
 > guide – Leverage build cache](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache)
 > for more information.
 
-## COPY --link
+### COPY --chown --chmod
+
+> **Note**
+>
+> Added in [Dockerfile 1.3](https://docs.docker.com/build/buildkit/dockerfile-frontend/).
```

Comment:
question: maybe we can remove this now?

Reply:
I think it's safe to remove n-2 release disclaimers

Reply:
done, removed em